build_oc.tool

```diff
@@ -261,7 +261,7 @@ package() {
     "Launchd.command"
     "Launchd.command.plist"
     "README.md"
-    "nvramdump"
+    "Legacy/nvramdump"
```

Comment:
But if you mean why doesn't the release have ./Legacy/nvramdump (Intel 10.6+)and ./nvramdump (Intel 10.9+ & ARM), then my feeling was that almost nobody who _uses_ this (as opposed to maybe developing it) will actually want the ARM version; and the install script for the NVRAM hook will copy and use whatever is in ./nvramdump position (at least, it currently will), so for almost all users it will be simpler to just ship the current 10.6 version, with manual step required from us, to recompile it, if the nvramdump.c code ever changes.